src/patternfly/components/Popover/docs/code.md

```diff
@@ -7,9 +7,9 @@ A popover is used to provide contextual information for another component on cli
 | Attribute | Applies to | Outcome |
 | -- | -- | -- |
 | `role="dialog"` | `.pf-c-popover` | Identifies the element that serves as the popover container. **Required**|
-| `aria-labelledby="[id value of .pf-c-popover__header-title]"` | `.pf-c-popover` | Gives the popover an accessible name by referring to the element that provides the dialog title. **Required when .pf-c-popover__header-title is present** |
-| `aria-label="[title of popover]"` | `.pf-c-popover` | Gives the popover an accessible name. **Required when .pf-c-popover__header-title is _not_ present** |
-| `aria-describedby="[id value of applicable content]"` | `.pf-c-popover` | Gives the popover an accessible description by referring to the popover content that describes the primary message or purpose of the dialog. Not used if there is no static text that describes the popover. |
+| `aria-labelledby="[id of .pf-c-popover__header-title]"` | `.pf-c-button` | Gives the element that triggers the popover an accessible name by referring to popover's title. **Required when .pf-c-popover__header-title is present** |
+| `aria-label="[title of popover]"` | `.pf-c-button` | Gives the element that triggers the popover an accessible name. **Required when .pf-c-popover__header-title is _not_ present** |
+| `aria-describedby="[id of .pf-c-popover__body]"` | `.pf-c-button` | Gives the element that triggers popover an accessible description by referring to the popover's body text, describing the primary message or purpose of the dialog. Not used if there is no static text that describes the popover. |
```

Comment:
I actually think that the previous descriptions for `aria-label`, `aria-labelledby`, and `aria-describedby` were correct. I was thinking that a popover is very similar to a modal, and so a lot of the aria attributes we are using for the modal would apply to the popover, too. And the previous version of these attributes were a direct copy from the modal example.

Please me know if you found other examples that suggest a different method for handling popovers.

Reply:
Ah yes, I believe you're right. I was considering tooltips and popovers to be similar in terms of a11y, but a popover is basically a modal. And the changes I made are how Bootstrap handles their popovers (with the `aria` attributes on the button that triggers it). I'll revert this file.